src/Plugin/Select/Handler.php

```diff
@@ -472,6 +494,7 @@ protected static function checkQueryForAliasSyntax(Payload &$payload, string &$q
 	 * @return TaskResult
 	 */
 	protected static function handleSelectDatabasePrefixed(Client $manticoreClient, Payload $payload): TaskResult {
+		self::checkQueryForAliasSyntax($payload);
```

Comment:
The current approach may not be very straightforward to understand, as it involves modifying the payload. Instead, we should consider either changing the naming or using an immutable type and reassigning it with the return value, rather than passing it by reference. This could help avoid potential confusion down the line. Alternatively, you may want to consider renaming the method to more accurately reflect that it not only checks but also modifies the payload.